File: iniciante/python/1000/1020.py
inicial = int(input())

ano, mes, dia = 0,0,0

if inicial >= 365:
    ano = inicial // 365
    inicial = inicial - ano * 365
if inicial >= 30:
    mes = inicial // 30
    inicial = inicial - mes * 30
if inicial < 30:
    dia = inicial

print(f'{ano} ano(s)')
print(f'{mes} mes(es)')
print(f'{dia} dia(s)')

File: iniciante/c/1020.c
#include <stdio.h>

int main(){
    int N;
    int ano = 0, mes = 0, dia = 0;
    scanf("%d", &N);
    if(N >= 365){
        ano = N / 365;
        N = N - ano * 365;
    }
    if(N >= 30){
        mes = N / 30;
        N = N - mes * 30;
    }
    dia = N;
    printf("%d ano(s)\n", ano);
    printf("%d mes(es)\n", mes);
    printf("%d dia(s)\n", dia);
    return 0;
}